cmd/ndm_daemonset/app/command/commands.go

```diff
@@ -49,22 +36,14 @@ func NewNodeDiskManager() (*cobra.Command, error) {
 		},
 	}
 
-	// add the glog flags
-	cmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
-	//add flag for /metrics endpoint port and endpoint path
-	cmd.PersistentFlags().String("port", ":9090", "Port to launch HTTP server.")
-	cmd.PersistentFlags().String("metricspath", "/metrics", "Endpointpath to get metrics.")
+	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
+	goflag.CommandLine.Parse([]string{})
```

Comment:
`CommandLine.Parse()` returns an error. It will not be checked, but we need to set it to `_` as a best practice.

Reply:
Addressed as part of https://github.com/openebs/node-disk-manager/pull/325/commits/22f399b3e89fecaaaca70c8f4d788c4aac188bd5